tests/queries/0_stateless/02706_show_columns.sql

```diff
@@ -55,6 +55,21 @@ CREATE TABLE NULL (c String) ENGINE = MergeTree ORDER BY c;
 SHOW COLUMNS FROM NULL;
 DROP TABLE NULL;
 
+DROP TABLE IF EXISTS `tab.with.dots`;
+CREATE TABLE `tab.with.dots`
```

Comment:
I guess the point of #69514 was to fix tables with dot(s) in their name. To test this and just this, we could keep the actual table definition dead-simple, i.e. we could have a simple string `c` (like in l. 54) and no skip index (l. 64).